src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -19,8 +19,9 @@
 
     @Override
     public CommandResult execute() {
+        addressBook.sort();
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        return new CommandResult(getMessageForPersonSortShownSummary(allPersons), allPersons);
```

Comment:
You could squash this commit with the previous two commits.
```
Messages: add sort command output message
``` 
...suggests that there's an output message that can be seen (but is not shown to the user).

```
Command: add method to call output message
```
...suggests that the newly added method is called to print the output message (but it doesn't).